src/data/communityCurationCohorts.json

```diff
@@ -192,6 +192,11 @@
             "id": 1974,
             "name": "CNRS",
             "tooltip": "The CNRS (National Center for Scientific Research)"
+          },
+          {
+            "id": 3831,
+            "name": "SSHOC",
+            "tooltip": "SSHOC"
```

Comment:
Tooltip should be `Social Sciences and Humanities Open Cloud`

Reply:
I've updated this now, thanks!